Monika After Story/game/script-topics.rpy

```diff
@@ -15413,3 +15413,75 @@ label monika_why_do_you_read:
     extend 3eua "While my situation in here has its downsides, [line_mid] opportunity to spend more time on the things I like."
     m 1ekbsu "...Though then again, nothing could ever beat spending more time with you~"
     return
+
+init 5 python:
+    addEvent(
+        Event(
+            persistent.event_database,
+            eventlabel="monika_tabletop_rping",
+            category=['monika','literature'],
+            prompt="Tabletop Roleplaying",
+            random=True,
+            aff_range=(mas_aff.NORMAL, None)
+        )
+    )
+
+label monika_tabletop_rping:
+    m "Say, [player], there's this literature genre that I've been meaning to bring up lately."
+    m "To be honest, I've been reading up on it while you've been gone, and it seems somewhat intriguing."
+    m "The genre is kind of unfamiliar territory to me though, but I thought I'd hear if you were interested in it."
+    m "It's called tabletop roleplaying! Have you heard of it?{nw}"
+    $ _history_list.pop()
+    menu:
+        m "It's called tabletop roleplaying! Have you heard of it?"
+        "Yes.":
+            m "Right, just please indulge me for a second then."
+
+        "No.":
+            m "Oh really? Then please indulge me while I tell you about it."
+
+    m "You may have played 'pretend' when you were a kid growing up, right?"
+    m "It's a game where you assume the role of a different person, maybe in another world, and act and speak as them to form a narrative."
+    m "When you're doing it with friends, it becomes much more fun, as you are also met with other people's character, and you interact with each other."
+    m "But 'pretend' always had one big problem{w=0.3}...What do you do when you can't agree on the story's plot?"
+    m "It could often devolve into a 'my character is invulnerable and always wins' kind of situation, you know?"
+    m "To counteract this problem, tabletop roleplaying was invented."
+    #Not sure if I should include this part about Dungeons and Dragons, since it's a pop culture reference, kind of. Though I think a topic about TRPGs isn't really complete without it either.
+    m "The most popular tabletop roleplaying game is called 'Dungeons & Dragons', or D&D, but that's far from the only one that exists, and its narrative form kind of always assumes a medieval fantasy setting."
+    m "In D&D, not all players are equal, since there necessarily always needs to be a 'Dungeon Master', basically someone who makes the setting of the story and controls the monsters and other obstacles."
```

Comment:
I'd cut the comma between 'equal' and 'since', and move the ending to a new line (this one is 200 characters too) : 
`m "In D&D, not all players are equal since there necessarily always needs to be a 'Dungeon Master'.`
`m "...Basically, the  'Dungeon Master' is the player who makes the setting of the story and controls the monsters and other obstacles."`